WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -528,7 +528,7 @@ private extension NotificationDetailsViewController {
 //
 private extension NotificationDetailsViewController {
     func attachSuggestionsViewIfNeeded() {
-        guard shouldAttachSuggestionsView else {
+        guard shouldAttachSuggestionsView, let suggestionsTableView = self.suggestionsTableView else {
             suggestionsTableView.removeFromSuperview()
```

Comment:
Note here! we should move the `guard let suggestionsTableView =...` unwrap above.

Why: `removeFromSuperview` might execute against the "non unwrapped" Suggestions TableView, leading to this crash:

<img width="1466" alt="Screen Shot 2020-11-16 at 11 54 03 AM" src="https://user-images.githubusercontent.com/1195260/99272447-946cb880-2806-11eb-8ee7-ff66f199f060.png">